[BUG] Benchmarks no longer run after refactor

### Observed behavior

The nightly benchmark GitHub action no longer succeeds after the refactor.

Here's the step that fails:

```
Run set -o pipefail; go test -bench=Bench -short -benchtime=5s -benchmem ./... | tee output.txt
pattern ./...: directory prefix . does not contain main module or its selected dependencies
Error: Process completed with exit code 1.
```
And a link to a failed run:

https://github.com/open-feature/flagd/actions/runs/4400751617/jobs/7706332843

### Expected Behavior

The benchmarks should run successfully.

### Steps to reproduce

_No response_

Here's the error output:
```console
Run set -o pipefail; go test -bench=Bench -short -benchtime=5s -benchmem ./core/... | tee output.txt
  set -o pipefail; go test -bench=Bench -short -benchtime=5s -benchmem ./core/... | tee output.txt
  shell: /usr/bin/bash -e {0}
  env:
    GO_VERSION: 1.19.[3](https://github.com/open-feature/flagd/actions/runs/4507101117/jobs/7934553491#step:4:3)
pattern ./core/...: directory prefix core does not contain main module or its selected dependencies
Error: Process completed with exit code 1.
```

--- a/.github/workflows/publish-benchmark.yaml
+++ b/.github/workflows/publish-benchmark.yaml
@@ -21,6 +21,9 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
 
+      - name: Workspace init
+        run: make workspace-init
+
       - name: Run benchmark
         run: set -o pipefail; go test -bench=Bench -short -benchtime=5s -benchmem ./core/... | tee output.txt
 
@@ -32,3 +35,7 @@ jobs:
           output-file-path: output.txt
           github-token: ${{ secrets.GITHUB_TOKEN }}
           auto-push: true
+          # Show alert with commit comment on detecting possible performance regression
+          alert-threshold: "130%"
+          comment-on-alert: true
+          fail-on-alert: false

